content/3_Silver/More_Prefix_Sums.mdx

```diff
@@ -652,6 +652,74 @@ for _ in range(query_num):
 
 <Problems problems="cum2" hideSuggestProblemButton />
 
+## Difference Arrays
+
+<Resources>
+	<Resource source="Codeforces" title="An Introduction To Difference Arrays" starred />
+</Resources>
```

Comment:
did you forget to link this resource? also probably why build is failing

Reply:
lmao